api/role-edit/index.ts

```diff
@@ -0,0 +1,55 @@
+import "../startup";
+import { Context, HttpRequest } from "@azure/functions";
+import { authorized, ApiRequestContext } from "../common/ApiRequestContext";
+import * as Validator from "validatorjs";
+import { badRequest, badRequestFor, ok } from "../common/http/CommonResults";
+import { RoleService } from "../common/services/RoleService";
+
+export default async function (context: Context, req: HttpRequest): Promise<void> {
+  await authorized(
+    context,
+    req,
+    "admin",
+    async ({ user }: ApiRequestContext) => {
```

Comment:
Like the comment on the other PR - is { user } used here? If it isn't, we're round-tripping to the Db for no reason.